docs/uuid.md

```diff
@@ -18,38 +20,8 @@ As your course is constructed, you may run into scenarios that require you to ch
 
 - **Renaming a directory: no new UUID needed.** For example if you rename a question or assessment directory, then you should leave the UUID the same, so that PrairieLearn will know that this is the same object but just with a new name.
 
-- **Copying a directory: needs a new UUID.** For example, if you copy an existing question or assessment to make a new one, then you will need a new UUID for the new object, so that PrairieLearn can distinguish it from the original.
-
-- **Moving a directory: needs a new UUID.** For example, if you move an assessment between course instances or if you move a question between courses, then you will need a new UUID for the object in its new location, so that PrairieLearn will know that this object should not be associated with the original.
-
-If you copy questions or assessments from a different course and don't give them new UUIDs, then a local development copy of PrairieLearn running on your own machine may not be able to detect that you have not changed the UUIDs. This will only be detected when you sync your course to the production PrairieLearn server, at which point it will give an error.
-
-If you incorrectly change the UUID in a `infoCourseInstance.json` file when you didn't mean to, then you will get a sync error like:
-
-```text
-Error: UUID 0339b989-d1f2-4f96-bcba-8717c68a64a9 from assessment exam1 in Sp18 already in use in different course instance (possibly in a different course)
-```
-
-An error like the one above can be fixed by reverting the UUID in the `infoCourseInstance.json` file back to its old value (which you can get from the `git` history).
-
-## Bulk addition of UUIDs
+- **Moving a directory: no new UUID needed.** For example, if you move an assessment between course instances or if you move a question between courses, you can leave the UUID as is. You can also rename it in this case, but it isn't necessary.
```

Comment:
A table format would be nice: one of what the action is, and a check/x if a UUID is needed.

Reply:
#12435